### DISCUSSION TITLE
How to export multiple entries as an object?

### DISCUSSION THREAD

**Discussion by goldingdamien** [webpack/webpack#16579] [Q&A]
I would like to export multiple entries as an object.
Example:

a.js
b.js
=>
{
 a: { ... },
 b: { ... },
}

I tried using output.libraryTarget = 'var', but it only exported the last entry point.
After a lot of time trying to find information on this, I found the below:

> When combining with the [output.library](https://webpack.js.org/configuration/output/#outputlibrary) option: If an array is passed only the last item is exported.

https://webpack.js.org/configuration/entry-context/#entry

This doesn't provide any help on what to do if you want to handle all the entry points.
What is the correct way of handling this?

Regards.

**Comment by alexander-akait** [comment]
I recommend to use multi compiler mode for such cases, we have an issue about it, but it was not design for it (but we mark it as a bug)
